app/views/management/schools/report.html.erb

```diff
@@ -14,7 +14,12 @@
 
 <% @overview_charts.each_with_index do |(energy, chart_config), index| %>
   <h2><%= t("management.schools.overview_charts.overview.#{energy.to_s}") %></h2>
-  <%= chart_tag(@school, chart_config[:chart], no_zoom: true, chart_config: {y_axis_units: select_y_axis(@school, chart_config[:chart], chart_config[:units])}) %>
+  <%= component 'chart',
```

Comment:
Does this need `axis-controls: false` as I don't see those controls on the current page (bottom of the dashboard, 'print view')

Reply:
:+1: